mk/dev.mk

```diff
@@ -310,6 +311,10 @@ dev/install/helm-docs: ## Bootstrap: Install helm-docs
 		&& set +x \
 		&& echo "helm-docs $(HELM_DOCS_VERSION) has been installed at $(HELM_DOCS_PATH)" ; fi
 
+.PHONY: dev/install/kube-lint
+dev/install/kube-lint:
+	go install golang.stackrox.io/kube-linter/cmd/kube-linter@v0.0.0-20220513142942-846f273ed465
```

Comment:
We push the version in vars at the top usually

Reply:
Good catch, already done. thanks.